docs/pbc/all/dynamic-multistore/202307.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -1,22 +1,89 @@
+![managing-stores](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/managing-stores.png)
 
-The capability consists of a few main components:
+## Creating a store
 
-* Dynamic store creation: stores can be created in the Back Office, moving the store entity from a fixed-configuration entity to a user-manageable entity.
+You can create a new store with a specified name. For this, in the Back Office, go to **Administration&nbsp;<span aria-label="and then">></span> Stores** page and click **Create Store**. This functionality simplifies the process of adding new stores to the existing setup.
 
-* Regional grouping: stores can be grouped into *regions*. This allows for further customization and can help organize stores by specific criteria like  geographical location.
+![creating-a-store](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/creating-a-store.png)
 
-* Infrastructure changes: Dynamic Multistore integrates with existing Jenkins jobs, Redis, and Elasticsearch configurations. It enables command line execution to be done per store or for all stores in a region-based setup.
+## Assigning locales
+
+When creating a store, you can assign locales and specify a default one for this new store, which ensures that the store is accessible and user-friendly for different regions.
+
+![assigning-locales](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/adding-locales.png)
+
+## Adding currencies
+
+The feature also allows the addition of currencies for the created store, ensuring that transactions can be conducted in various currencies as needed.
+
+![adding-currencies](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/adding-currencies.png)
+
+## Publishing and synchronizing
+
+When the new queue infrastructure is published, the store becomes visible on the storefront.
+
+## Assigning products and prices
+
+You can assign products and their prices to a product using the Back Office UI or other methods of creating a new product in the system. This offers flexibility in managing the product catalog.
+
+The following image shows how you can assign a product to an already existing store.
+
+![store-relation](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/store-relation.png)
+
+The following demonstrates shows how you can assign a product price to a specific store.
+
+![product-prices](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/product-prices.png)
+
+## Viewing the store on the storefront
+
+The newly created store is viewable on the storefront, complete with the assigned products and prices.
+
+The following example demonstrates the created store from the Back Office being in the store switcher.
```

Comment:
Once the store is published on the Storefront, customers can select it in the store switcher.